app/pages/cif/contact/form/[contactForm].tsx

```diff
@@ -120,6 +120,9 @@ function ContactFormPage({ preloadedQuery }: RelayProps<{}, ContactFormQuery>) {
           justify-content: space-between;
           align-items: start;
         }
+        header h2 {
+          padding-right: 10px;
+        }
```

Comment:
Let's add the same thing to the contact view page (`app/pages/cif/contact/[contact].tsx`)

Reply:
Added - see update.